sensor_msgs/Image image
string source
float32 score
string object_id
string detector
geometry_msgs/Point topLeft
geometry_msgs/Point bottomRight
geometry_msgs/Point gaze
